.github/workflows/build_and_test.yaml

```diff
@@ -87,7 +87,7 @@ jobs:
           restore-keys: ${{ runner.os }}-gradle
       - name: Run CheckStyle and FindBugs
         run: |
-          ./gradlew --no-daemon javadoc findbugsMain checkstyleMain checkstyleTest checkstyleJmh 
```

Comment:
Automatic code formatter fixed the trailing whitespace. I figured I'd leave it for general code consistency